src/main/kotlin/minesweeper/ResultView.kt

```diff
@@ -0,0 +1,37 @@
+package minesweeper
+
+import minesweeper.domain.Cells
+
+const val CELL = "#"
```

Comment:
`ResultView` 외에 사용하는 곳이 없어 보이네요. public이 아닌 private으로 바꾸면 어떨까요?

```suggestion
private const val CELL = "#"
```